docs/CaliptraReleaseChecklist.md

```diff
@@ -0,0 +1,52 @@
+![OCP Logo](./images/OCP_logo.png)
+
+<p style="text-align: center;">Caliptra Release Checklist</p>
+
+<p style="text-align: center;">Version 1.1</p>
+
+<div style="page-break-after: always"></div>
+
+# Overview
+
+This document provides the signoff checklist that is used when finalizing any Caliptra release version.
+
+# Release Creation
+
+## Versioning
+
+Caliptra RTL releases may be created for new major, minor, or patch versions, as described in the [semantic versioning specification](https://semver.org/spec/v2.0.0.html). The version number is reflected in the register CPTRA_HW_REV_ID. Steps described in this document are followed for each of these releases.
+
+## Branches
+
+Each major and minor release is created as a tag on the branch `main` of the caliptra-rtl repository. The tag is created using GitHub's repository release tagging feature, which also generates a zip file containing all of the code and documentation for that release. After tagging the release, any subsequent commits to `main` are pursuant to development efforts on future release versions, so the tagged release must be used to download the official release code.
+
+When necessary, a patch release may be applied retroactively to earlier versions of Caliptra. In this case, a new branch is created to contain the patched code base. Patch release branches will follow the naming convention `patch_v<MAJOR>.<MINOR>` to indicate which version is being patched. Once the patched branch has reached it's finalized state, a tag is created on the patch branch to indicate the full version number of that patch. Thus, any patch release will be created as a tag on the same branch, of an incrementing patch version number.
```

Comment:
minor edits for this para for grammar and present tense. Also please verify that edits in last sentence preserve accuracy: 

When necessary, a patch release may be applied retroactively to earlier versions of Caliptra. In this case, a new branch is created to contain the patched code base. Patch release branches follow the naming convention, `patch_v<MAJOR>.<MINOR>`, to indicate which version is being patched. After the patched branch reaches its finalized state, a tag is created on the patch branch to indicate the full version number of that patch. Thus, any patch release is created as a tag on the same branch, with an incrementing patch version number.

Reply:
Done